src/components/AudioAnalysis/SectionChart.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { PureComponent } from 'react';
```

Comment:
@Mattallmighty i think this is an old module (jsx) and there is already a tsx duplicate.
shall we remove this file?